src/lib/urlParser.js

```diff
@@ -33,13 +33,13 @@ const participantRegex = new RegExp(`${Config.HOSTNAME}/participants/([A-Za-z0-9
 const transfersRegex = new RegExp(`${Config.HOSTNAME}/transfers/([a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12})`, 'i')
 const participantTransfersRouteRegex = new RegExp(/\/participants\/([A-Za-z0-9_]*)\/transfers/, 'i')
 
-const nameFromParticipantUri = (uri, callback) => {
+const nameFromParticipantUri = (uri, done) => {
```

Comment:
Why did you change the name from call back to done for all of these? What does done mean?